src/metadata-stream.js

```diff
@@ -6,12 +6,24 @@
 (function(window, videojs, undefined) {
   'use strict';
   var
```

Comment:
Those are all great rules by themselves, but I think the combination makes for a breadcrumb trail made of commas and pain. For what it's worth, I really like Ben Alman's [take on multiple var statements](http://benalman.com/news/2012/05/multiple-var-statements-javascript/). Feels roughly similar to where we've landed in VJS core?

> My rule is as-follows: I’ll put multiple comma-separated declarations in a single var statement, but only if they don’t have assignments. Assignments must be in their own var statement. Like this:
```javascript
var foo, bar, baz;
var abc = 1;
var def = 2;
```

Again, this probably isn't the place for this conversation, but as someone not familiar with this style, this is ripe for massive wtf moments. Anecdotally, I had one of those massive wtf moments when trying to help someone out internally with BCC code that uses this same style.

Somewhat related: are we planning on using [standard](https://github.com/videojs/standard) across contrib plugins as well?

`/me climbs off soap box`